grpc/src/main/java/io/stargate/grpc/payload/cql/ValuesHandler.java

```diff
@@ -35,17 +35,22 @@
 import io.stargate.grpc.codec.cql.ValueCodec;
 import io.stargate.grpc.codec.cql.ValueCodecs;
 import io.stargate.grpc.payload.PayloadHandler;
-import io.stargate.proto.QueryOuterClass.*;
-import io.stargate.proto.QueryOuterClass.Payload.Type;
+import io.stargate.proto.QueryOuterClass.ColumnSpec;
+import io.stargate.proto.QueryOuterClass.QueryParameters;
+import io.stargate.proto.QueryOuterClass.ResultSet;
+import io.stargate.proto.QueryOuterClass.Row;
+import io.stargate.proto.QueryOuterClass.TypeSpec;
+import io.stargate.proto.QueryOuterClass.Value;
+import io.stargate.proto.QueryOuterClass.Values;
 import java.nio.ByteBuffer;
 import java.util.ArrayList;
 import java.util.List;
 
 public class ValuesHandler implements PayloadHandler {
   @Override
-  public BoundStatement bindValues(Prepared prepared, Payload payload, ByteBuffer unsetValue)
+  public BoundStatement bindValues(Prepared prepared, Any payload, ByteBuffer unsetValue)
```

Comment:
Nit, If this will be used from the Scala language, there will be a clash with the Scala syntax. `Any` is equivalent of Java's `Object`

Reply:
Heh, I'll make it fully qualified if we start using it from Scala. :)